README.md

```diff
@@ -1 +1,95 @@
-# laravel-transactional-event-publisher
\ No newline at end of file
+Laravel Transactional Event Publisher
+=====================================
+
+[![Latest Version](https://img.shields.io/github/release/softonic/laravel-transactional-event-publisher.svg?style=flat-square)](https://github.com/softonic/laravel-transactional-event-publisher/releases)
+[![Software License](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=flat-square)](LICENSE.md)
+[![Build Status](https://img.shields.io/travis/softonic/laravel-transactional-event-publisher/master.svg?style=flat-square)](https://travis-ci.org/softonic/glaravel-transactional-event-publisher)
+[![Coverage Status](https://img.shields.io/scrutinizer/coverage/g/softonic/laravel-transactional-event-publisher.svg?style=flat-square)](https://scrutinizer-ci.com/g/softonic/laravel-transactional-event-publisher/code-structure)
+[![Quality Score](https://img.shields.io/scrutinizer/g/softonic/laravel-transactional-event-publisher.svg?style=flat-square)](https://scrutinizer-ci.com/g/softonic/laravel-transactional-event-publisher)
+[![Total Downloads](https://img.shields.io/packagist/dt/softonic/laravel-transactional-event-publisher.svg?style=flat-square)](https://packagist.org/packages/softonic/laravel-transactional-event-publisher)
+
+Laravel package to handle atomicity between Eloquent model operations and event message generation and storing linked to this model operation. Main features:
+
+* Atomic transaction between Eloquent model operation, event generation and sent.
+* Events sent to a AMQP system
+
+Documentation
+-------------
+
+To add the Observer generates and sends to the event Store just add it in the `boot()` method of `AppServiceProvider` class
+
+```
+...
+class AppServiceProvider extends ServiceProvider
+{
+    public function boot()
+    {
+        ModelToSendEvents::observe(\Softonic\TransactionalEventPublisher\ModelObserver::class);
```

Comment:
Shouldn't this be done in the package boot?

Reply:
Is an example to attach the model observer